.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.{jade,pug}]
```

Comment:
I'm a bit surprised to still see jade/pug in a new project

Reply:
https://github.com/cozy/create-cozy-app/issues/1292